tiledbcontents/tiledbcontents.py

```diff
@@ -481,7 +481,7 @@ def _write_bytes_to_array(
             tiledb_uri, final_array_name = self._create_array(tiledb_uri, 5)
 
         with tiledb.open(tiledb_uri, mode="w", ctx=TILEDB_CONTEXT) as A:
-            A[range(len(contents))] = {"contents": contents}
+            A[0:len(contents)] = {"contents": contents}
```

Comment:
Should this be from `0:len(contents)-1`? Or is this write exclusive on the upper bound?

Reply:
This is the only arrangement that worked, debugged the query in rest server and if not exclusive it replied with too many bytes